### PR TITLE
docs: Update clustering settings

### PR DESCRIPTION
**Pull request by paultranvan**
Some settings changed after some improvements in the clustering service

### CODE REVIEW

**Review by ptbrowne — APPROVED** [review]

File: docs/io.cozy.photos.md [hunk 1]
```diff
@@ -22,9 +22,8 @@ This doctype is used to store settings for photos, useful for the clustering.
 
 ### Clustering settings
 
-- `lastDate`: {date} the date of the last clustered photo (based on `created_at`).
+- `lastDate`: {date} the date of the last clustered photo (based on `cozyMetadata.createdAt`).
```
Comment (Crash--) [inline docs/io.cozy.photos.md:25]: based on `cozyMetadata.createdAt` of what? Of the last clustered photo io.cozy.file document?

Comment (paultranvan) [reply docs/io.cozy.photos.md:25]: Yes, is there a possible confusion? I don't see what else it could be :thinking:

Comment (paultranvan) [reply docs/io.cozy.photos.md]: I rephrased to remove any possible confusion ;)

Comment (Crash--) [reply docs/io.cozy.photos.md]: Me neither, but it was to be 100% sure.

**Review by Crash-- — APPROVED** [review]

**Pull request merged**